include/dlaf/init.h

```diff
@@ -43,14 +43,13 @@ struct configuration {
   std::size_t umpire_host_memory_pool_alignment_bytes = 16;
   double umpire_host_memory_pool_coalescing_free_ratio = 1.0;
   double umpire_host_memory_pool_coalescing_reallocation_ratio = 1.0;
-  std::size_t umpire_device_memory_pool_initial_block_bytes = 1 << 30;
-  std::size_t umpire_device_memory_pool_next_block_bytes = 1 << 30;
+  std::size_t umpire_device_memory_pool_initial_block_bytes = 1 << 16;
+  std::size_t umpire_device_memory_pool_next_block_bytes = 1 << 16;
```

Comment:
@biddisco word of warning in case you benchmark on todi: having them this small really kills performance. As an example, the eigensolver on a 10k input matrix goes from ~3 seconds to ~9 seconds on two GPUs.